x-pack/plugins/cases/public/components/all_cases/assignees_filter.test.tsx

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
Could you please add a test where we test if the list of users is sorted and the current user is on the top?